eng/testing/linker/SupportFiles/Directory.Build.props

```diff
@@ -3,6 +3,7 @@
     <SkipConfigureTrimming>true</SkipConfigureTrimming>
     <UseDefaultBlazorWASMFeatureSwitches>false</UseDefaultBlazorWASMFeatureSwitches>
```

Comment:
You can now remove all these flag setting to false since they are now turned off by default because the condition changed.

Reply:
Don't we need to have them here explicitly in order to know from where this settings come?

Reply:
I don't think so. For flags where it's default value is false, we usually don't set them to false in any place.

Reply:
Removed, thanks.